ui/pages/settings/security-tab/security-tab.component.js

```diff
@@ -1108,7 +1076,7 @@ export default class SecurityTab extends PureComponent {
           {this.context.t('metrics')}
         </span>
         <div className="settings-page__content-padded">
-          {this.renderMetaMetricsOptIn()}
```

Comment:
Was this change needed for this PR? Or is it just an improvement of the codebase?

Reply:
@danjm  in the hook I call, I execute a logic whereby if Profile Sync is not active and metametrics is not active, I log out the user. Conversely, if the user is logged out and activates metametrics, I log in the user. This function is necessary for the next implementation of notifications (next PR).